test/apiv2/10-images.at

```diff
@@ -196,12 +196,16 @@ fi
 curl -XPOST --data-binary @<(cat $CONTAINERFILE_TAR) \
     -H "content-type: application/tar" \
     --dump-header "${TMPD}/headers.txt" \
-    -o /dev/null \
-    "http://$HOST:$PORT/v1.40/libpod/build?dockerfile=containerfile" &> /dev/null
+    -o "${TMPD}/response.txt" \
+    "http://$HOST:$PORT/v1.40/libpod/build?dockerfile=containerfile&q=true" &> /dev/null
 if ! grep -q '200 OK' "${TMPD}/headers.txt"; then
     echo -e "${red}NOK: Image build from tar failed response was not 200 OK (application/tar)"
     BUILD_TEST_ERROR="1"
 fi
+if grep -q 'Successfully built ' "${TMPD}/response.txt"; then
```

Comment:
I do not like negative checks. There could still be other messages in the output which should not be printed with quiet.
I think it would make more sense to only match that the output is a valid ID.

Reply:
That's a good idea, thanks